students/chris_lombardi/Lesson03/mailroom.py

```diff
@@ -0,0 +1,115 @@
+#UWPCE PY210
+#Lesson03, Mailroom Part 1
+import sys
+
+DONOR_LOG = [] #Log of donors and their respective donation history.
```

Comment:
capitals are used for constants, since your DONOR_LOG is mutable it should be defined in all lower case.